webclient/src/services/TimeService/TimeService.ts

```diff
@@ -0,0 +1,33 @@
+import { DateTime } from 'luxon';
+
+class TimeService {
+  renderAccountAge(seconds: number) {
```

Comment:
Does this support `< 1 Day`? I dont think we care about minutes/hours/seconds for account age, tho i think game age starts with `< 1 minute`.

Reply:
Currently, for less than one days, no text is returned. I am adding a test case to show this